xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -574,7 +574,7 @@ struct AppModel: ModelProtocol {
         self.databaseMigrationStatus == .succeeded
     }
     
-    var selectedAppTab: AppTab = .feed
+    var selectedAppTab: AppTab = .notebook
```

Comment:
I think the app should actually default to the .notebook view on first run, because the feed is always empty.